bin/test

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+require "fileutils"
+
+APP_ROOT = File.expand_path("..", __dir__)
+
+# Run coverage checks during rspec
+ENV['COVERAGE'] = '1'
+
+# Required for the DB down/up test
+ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK'] = '1'
+
+# Test commands (to be exec'd in order)
+TESTS = [
+  %w(rspec -f html),
+  %w(rubocop),
+  %w(rails --trace db:drop db:create db:migrate),
+]
+
+FileUtils.chdir APP_ROOT do
+  exit TESTS.reduce(true) { |passfail, cmd| system(*cmd) && passfail }
```

Comment:
@anarchivist How's this for parity with what came before? It runs all of the same tests sequentially, exiting 0 if they all succeed and 1 otherwise.

Reply:
lgtm, thanks!